aws/resource_aws_rds_global_cluster.go

```diff
@@ -498,3 +518,95 @@ func waitForRdsGlobalClusterRemoval(conn *rds.RDS, dbClusterIdentifier string) e
 
 	return nil
 }
+
+func resourceAwsRDSGlobalClusterUpgradeMajorEngineVersion(clusterId string, engineVersion string, conn *rds.RDS) error {
+	input := &rds.ModifyGlobalClusterInput{
+		GlobalClusterIdentifier: aws.String(clusterId),
+	}
+	input.AllowMajorVersionUpgrade = aws.Bool(true)
+	input.EngineVersion = aws.String(engineVersion)
+	err := resource.Retry(5*time.Minute, func() *resource.RetryError {
```

Comment:
I think current best practice is to create a `const` in `internal/service/rds/waiter/waiter.go` for these timeout times.